app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1832,7 +1832,8 @@ public void webViewFailedLoading(String url) {
 
   @Override
   public void webViewProgressChanged(int progress) {
-    progressBar.setProgress(progress);
+    if (progressBar != null)
```

Comment:
Use curly brackets here so that more statements can be added in future easily without many changes.